torch/utils/dlpack.py

```diff
@@ -25,3 +15,38 @@
 The dlpack shares the tensors memory.
 Note that each dlpack can only be consumed once.
 """)
+
+
+def from_dlpack(ext_tensor: Any) -> torch.Tensor:
+    """from_dlpack(ext_tensor) -> Tensor
+
+    Convers a tensor from a external library into a ``torch.Tensor``
```

Comment:
"Converts a DLPack capsule into a tensor."

The phrase "DLPack capsule" should be a link to the DLPack documentation

Reply:
This comment still needs to be resolved

Reply:
> The phrase "DLPack capsule" should be a link to the DLPack documentation

There's no such thing yet. DLPack itself only has comments in [dlpack.h](https://github.com/dmlc/dlpack/blob/main/include/dlpack/dlpack.h). `__dlpack__` is part of the array API standard, see [here](https://data-apis.org/array-api/latest/API_specification/array_object.html#dlpack-self-stream-none).

Reply:
I am fine with adding the link to the API standard, would it be possible to do it in another PR?
Thanks!